### ISSUE TITLE
Dialog: When using screen reader, the content of the dialog is getting read twice

### ISSUE THREAD

**Issue by rinnuagna** [primefaces/primereact#8129]
### Describe the bug

The screen reader reads the content twice and even header and other contents

### System Information

```Shell
primereact: 10.9.1
browser: chrome
```

### Steps to reproduce the behavior

1. Open screen reader.
2. Click on button to open dialog
3. check the screen reader

### Expected behavior

It should be only read once, unnecessarily adding things in reader

**Comment by rinnuagna** [comment]
<img width="1757" height="872" alt="Image" src="https://github.com/user-attachments/assets/82c380c6-8344-4157-ad3b-0cd9e9ba0125" />

**Comment by melloware** [comment]
duplicate https://github.com/primefaces/primereact/issues/7974

**Comment by melloware** [comment]
actually not quite a duplicate.
